packages/light-client/test/utils/getGenesisData.ts

```diff
@@ -10,7 +10,7 @@ import {NetworkName} from "@lodestar/config/networks.js";
 
 /* eslint-disable no-console */
 
-const networksInInfura: NetworkName[] = ["mainnet", "goerli"];
+const networksInInfura: NetworkName[] = ["mainnet" /*"goerli"*/];
```

Comment:
there are bunch of Infura related test cases that still reference goerli, needs to be cleaned up separately